doc/api/API.md

```diff
@@ -52,20 +52,20 @@ This section is for viewer related non-static methods. They would be callable in
 Plugin:
 ```dart
 void showViewer() async {
-  PdftronFlutter.openDocument('https://pdftron.s3.amazonaws.com/downloads/pl/PDFTRON_about.pdf');
+  await PdftronFlutter.openDocument('https://pdftron.s3.amazonaws.com/downloads/pl/PDFTRON_about.pdf');
 }
 ```
 
 Widget (DocumentViewController):
 ```dart
-void _onDocumentViewCreated(DocumentViewController controller) {
-    controller.openDocument('https://pdftron.s3.amazonaws.com/downloads/pl/PDFTRON_about.pdf');
+void _onDocumentViewCreated(DocumentViewController controller) async {
+  await controller.openDocument('https://pdftron.s3.amazonaws.com/downloads/pl/PDFTRON_about.pdf');
 }
 ```
 
 We suggest that you stick with either version for the APIs that are callable in both versions, to avoid unnecessary problems.
```

Comment:
I don't quite understand this sentence. Do you mean choose either widget or plugin, but not both?